product_docs/docs/pem/8/installing/prerequisites.mdx

```diff
@@ -5,97 +5,117 @@ redirects:
 - /pem/8/installing_pem_server/pem_server_inst_linux/prerequisites/
 ---
 
-To install a Postgres Enterprise Manager server on Linux, you may need to perform some preliminary configuration:
+You can install PEM on a single server, or you can install the web application server and the backend database on two separate servers. You must prepare your servers for PEM installation. 
+
+After completing the prerequisites, [install](/pem/8/installing/linux_x86_64/index.mdx) and [configure](/pem/8/installing/configuring_the_pem_server_on_linux.mdx) PEM. If you're using two servers, install and configure PEM on both servers.
+
+To install a PEM server on Linux, perform this preliminary configuration:
+    
+1.  Install a [supported Postgres instance](/pem/8/index/#postgres-compatibility) for PEM to use as a backend database. 
+
+You can install this instance on the same server as you plan to use for the PEM web application or on a separate server. You can also use an existing Postgres instance, providing you meet the conditions that follow.
```

Comment:
I didn't know which conditions you are referring to. Can you clarify?

Reply:
The conditions specified from bullet no 2 onwards